templates/web/base/report/_item_small.html

```diff
@@ -30,3 +30,11 @@
     [% ELSIF problem.bodies_str_ids.size == 0 %] [% loc('(not sent to council)') %]
     [% END %]
 [% END %]
+<div class='assigned-to' style='margin-top:0.25em'>
+[%  IF c.user.has_permission_to('report_inspect', c.cobrand.body.id ) %] 
```

Comment:
other permission checks on this page use `problem.bodies_str_ids`.

Thinking aloud - what if a user has the assign permission but not the report_inspect permission? You can assign but not see assignments? Do we want to make it work, or just assume anyone with the former permission will also have the latter.